src/modules/status-checker.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable object-shorthand */
+const tasksArr = JSON.parse(localStorage.getItem('tasksArr')) || [];
+
+function checker(state) {
+  if (state) {
+    return true;
+  }
+  return false;
+}
```

Comment:
- Kindly Make sure to use arrow functions in order to follow [JavaScript best practices](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/exercises/clean_code_self_assessment.md)